Google.Api.Gax.Grpc.GrpcNetClient.Tests/Google.Api.Gax.Grpc.GrpcNetClient.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
tiny nit: Is this changing EOL?

Reply:
More likely to be a BOM introduced or removed. But this project will be going away in the next PR anyway, so I won't bother changing it :)